### DISCUSSION TITLE
Removing a horizontal white line from the middle of an image

### DISCUSSION THREAD

**Discussion by feketegabor** [ImageMagick/ImageMagick#5391] [Help]
I want to do ocr, but there is a mostly white horizontal rectangle separating the image in a way, that the characters are split into two parts. How can I delete this white line from the image, and push the upper and lower remaining parts together? 

I need a way to identify where this white line is, then remove it, and stich together the image.  The line is in a slightly different position on every image. The white line doesn't cover any data, it just separates the image, it's the printer's fault. The white line unfortunately is not consistently solid white, so i need a way to identify the line. If a row is mostly white, or very close to it in the middle of the image, with no black and dark grey pixels in it, I would like to remove it from the image.

**Comment by fmw42** [comment]
What is your IM version and platform?  Can you post an example image?
